src/Adapter/GeneralConfiguration.php

```diff
@@ -44,24 +45,38 @@ class GeneralConfiguration implements DataConfigurationInterface
      */
     private $cookie;
 
-    public function __construct(Configuration $configuration, Cookie $cookie)
+    /**
+     * @var bool
+     */
+    private $isDebug;
+
+    public function __construct(Configuration $configuration, Cookie $cookie, bool $isDebug)
```

Comment:
It's a BC, the class is not in final.

Reply:
Correct. I labeled the PR as "BC Break" and documented the BC.

Reply:
You can avoid the bc breaks, if you add a default value :wink: